docs/cli-documentation.md

```diff
@@ -2,8 +2,9 @@
 
 ## Disclaimer
 
-The project is in **alpha** version.
-Readers can refer to our [opened GitHub issues](https://github.com/airbytehq/airbyte/issues?q=is%3Aopen+is%3Aissue+label%3Aarea%2Foctavia-cli) to check the ongoing work on this project.
+The Octavia CLI is an alpha, unofficial CLI that won't be maintained. <br>
```

Comment:
Maybe a 
```
:::warning
The Octavia CLI is alpha and...
:::

wdyt?

Reply:
Adding

Reply:
Will render like this: 
![CleanShot 2023-08-25 at 11 46 15@2x](https://github.com/airbytehq/airbyte/assets/7691812/b538e431-2e20-4f44-b472-7bf00fa5b395)